test/bases/renku_data_services/data_api/test_session_secrets.py

```diff
@@ -0,0 +1,367 @@
+"""Tests for session secrets blueprint."""
+
+from typing import Any
+
+import pytest
+from sanic_testing.testing import SanicASGITestClient
+from ulid import ULID
+
+from renku_data_services.users.models import UserInfo
+from test.bases.renku_data_services.data_api.utils import merge_headers
+
+
+@pytest.fixture
```

Comment:
Ralf just merged his PR about the tests improvements. But now all fixtures should be async. Otherwise the load order of fixtures gets weird. Because I think pytest has weird rules about how it loads async vs sync fixtures.

Reply:
Done in 06a08030c0fa9cfadfa5121259854cd3533aee96.